Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md

```diff
@@ -0,0 +1,139 @@
+# Introduction
+
+When starting to learn React, the beginners come across a few frequently used words in a syntactical form in React examples _import_, _export_, _modules_, and many more. Many questions might arise in one of the React learning brains some of which may be :
+
+1. What do these terms mean and what makes them so necessary in React?
+
+2. What aspects of React make them necessary.
+
+_Through this documentation I would like to explain these concepts in much more detail to all those beginners out there seeking to know more about different aspects of these new terms._
+
+## Import and Export
+
+The plain, simple English dictionary meaning of the words **import** and **export** are roughly defined as
+> to bring in/out across the border
+
+Similarly, these words explain a kind of **border-crossing** in the land of JavaScript.
+
+Whenever you set up a basic react project you are provided with a _boilerplate_ for your project and henceforth you start building your project on it.
+
+<div style="display: flex; justify-content: center;">
+    <img src="./assets/ReactSetup.png" style="height:   300px; width: auto; ">
```

Comment:
same here also for image

Reply:
In my forked repo I can see the images in the preview of the documentation.
<br>
![image](https://user-images.githubusercontent.com/56582834/134342955-eae8186a-f16a-4ab6-b3a2-ad9fe8de808a.png)